josh-mysql/README.md

```diff
@@ -0,0 +1,39 @@
+# MySQL Provider for JOSH
+
+The MySQL provider uses mysql for persistent storage of JOSH data.
+
+## Installation
+
+I won't do a tuto yet on installation. There is lots of them on the internet, so google it.
+
+## Usage
+
+Mysql providers don't create connection for you. You must create it manually like that :
+
+```js
+const Josh = require('@joshdb/core');
+const JoshMySQL = require('@joshdb/mysql');
+const { createConnection } = require("mysql");
+
+const connection = createConnection({
+  host: "localhost",
+  user: "root",
+  password: "pAsSwOrD1234",
+  database: "mydb"
+})
+
+const db = new Josh({
+  name: 'testing',
+  provider: JoshMySQL,
+  providerOptions: {
+    connection: connection
+  }
+});
```

Comment:
In my opinion the user shouldn't have to supply the connection to Josh, instead the credentials should be passed in the providerOptions and the provider should create the connection.

Reply:
Kay. I did that to allow people use the same connection for multiple JOSHs

Reply:
Hmm, couldn't you do both?